packages/storage/amplify_storage_s3_dart/test/storage_s3_service/storage_s3_service_test.dart

```diff
@@ -402,7 +369,7 @@ void main() {
 
         expect(
           storageS3Service.list(
-            path: 'a path',
+            path: const StoragePath.fromString('apath'),
```

Comment:
I made "a path" one word because I thought it couldn't have a space in it. I'm not sure if this was on purpose or not or if I should change it back

Reply:
```suggestion
            path: const StoragePath.fromString('a path'),
```